### PR TITLE
Add 508 Markdown Checklist

### LINKED ISSUE #33: 508-checklist.md
**Is your feature request related to a problem? Please describe.**
Currently, the ospo-guide does not contain guidance on 508ing our powerpoint presentations

**Describe the solution you'd like**
508 checklist document has been completed. We are ready to ship to ospo-guide in markdown.
Let's name the file: 508-checklist.md

### PR DESCRIPTION
**Pull request by DinneK**
closes #33 

## Problem
Slide presentations do not have a cohesive set of checks to ensure they comply with 508 standards. 

## Solution
Create a markdown file with a comprehensive set of checks.

## Result
A checklist has been created to ensure checks are adhered to.

## Test Plan
Test manually and check with markdown previewer.

## Next Steps
Add checks for PDF files and PDF testing.

### CODE REVIEW

File: outbound/508-checklist.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| 11 | [Ensuring Descriptions of Embedded Audio, Video and Multimedia Files are Accurate](https://www.section508.gov/training/presentations/aed-cop-pptx11/) | <ul><li>Carefully name your media files with descriptive titles that clearly convey the content, and when necessary, add detailed captions or alternative text to explain what is being shown or heard.</li><li>Include key details like speaker names, topic, or location when relevant.</li><li>When adding images with embedded audio or video, provide alternative text that describes what is visually happening in the image alongside the audio or video content.</li></ul> | |
+| 12 | [Excluding Flashing Objects](https://www.section508.gov/training/presentations/aed-cop-pptx12/) | <ul><li>Remove flashing objects from slides since they can cause seizures</li</ul>
+| 13 | [Saving in the .pptx Format with a Descriptive Filename](https://www.section508.gov/training/presentations/aed-cop-pptx13/) | <ul><li>Use a descriptive file name to identify the document and its purpose so everyone can locate, open, and switch between documents.</li></ul> | |
```
Comment (decause-gov) [inline outbound/508-checklist.md:29]: Something that we often do is

| 14 | [Export the .pptx to a .pdf](https://www.section508.gov/create/pdfs/) | <ul><li>PDF files typically look the same on every device, and don't rely on dynamically loading things like typefaces or images from local systems. PDFs help preserve formatting, and limit usage of features like animations or transitions, that can be difficult to process in screenreaders or other accessibility tools.</li></ul> | |

**Review by decause-gov — APPROVED** [review]
- Where we don't need to specify Powerpoint, let's be more generic.
- Added an extra bullet to the Google Slides section about exporting to .pdf. (PDF export is mentioned already in the powerpoint specific section.)

This is *excellent work* very exciting! Once comments are resolved, LGTM +1 🚢

File: outbound/508-checklist.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Powerpoint Presentation 508 Checklist
```
[the remark this answers is not in the document]
Comment (DinneK) [reply outbound/508-checklist.md:1]: Changed to "Digital Presentation"

File: outbound/508-checklist.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## About
+
+508 compliance refers to the standards outlined by [Section 508 of the Rehabilitation Act](https://www.access-board.gov/about/law/ra.html), which mandates that electronic documents must be accessible to all, including people with disabilities. The OSPO seeks to make all our PowerPoint presentations, documents, and other electronic materials fully accessible and compliant with 508 standards.
```
[the remark this answers is not in the document]
Comment (DinneK) [reply outbound/508-checklist.md:5]: Removed.

File: outbound/508-checklist.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Below is our process making a powerpoint slide presentation 508 compliant.
```
[the remark this answers is not in the document]
Comment (DinneK) [reply outbound/508-checklist.md:7]: Changed to "Below is our process for making a slide presentation accessible:"

File: outbound/508-checklist.md (hunk 1 above)
Comment (DinneK) [reply outbound/508-checklist.md:29]: 14 added

**Pull request merged**